---
name: Arnold Kuzniar
nlescWebsite: https://www.esciencecenter.nl/profile/dr.-arnold-kuzniar
researchgateUrl: https://www.researchgate.net/profile/Arnold_Kuzniar
linkedInUrl: https://www.linkedin.com/in/arnold-kuzniar-06312210
githubUrl: https://github.com/arnikz
jobTitle: eScience Research Engineer
affiliation:
- http://software.esciencecenter.nl/organization/nlesc

photo: https://www.esciencecenter.nl/img/team/Arnold-Kuzniar-web.jpg

contactPersonOf:
- http://software.esciencecenter.nl/software/fairdatapoint
- http://software.esciencecenter.nl/software/piqmie
- http://software.esciencecenter.nl/software/netclust

ownerOf:
- http://software.esciencecenter.nl/software/fairdatapoint
- http://software.esciencecenter.nl/software/piqmie
- http://software.esciencecenter.nl/software/netclust

contributorOf:
- http://software.esciencecenter.nl/software/fairdatapoint
- http://software.esciencecenter.nl/software/piqmie
- http://software.esciencecenter.nl/software/netclust

userOf:
- http://software.esciencecenter.nl/software/fairdatapoint
- http://software.esciencecenter.nl/software/piqmie
- http://software.esciencecenter.nl/software/netclust

engineerOf:
- http://software.esciencecenter.nl/project/odex4all
- https://www.esciencecenter.nl/project/prediction-of-candidate-genes-for-traits-using-interoperable-genome-annotat
---
Arnold graduated in Molecular Biology (2004, Comenius University in Bratislava, Slovakia), with a specialization in bioinformatics. In the last year of his masters, he received a scholarship to study at the Wageningen University and Research Centre (WUR), the Netherlands. He developed bioinformatics tools to efficiently map the genetic changes and to visualize epidemiologically relevant biomarkers in newly sequenced genomes of White spot syndrome (shrimp) virus isolates.

In the same year, Arnold started his PhD research at the Laboratory of Bioinformatics, WUR, on improving orthology-based detection in fully sequenced genomes and the integration of protein orthology/family resources. In close collaboration with SURFsara, he used the Dutch Life Science Grid to compute a comprehensive map of corresponding (orthologous) proteins across available proteomes from all three domains of life. Analyzing this large data set required a different approach as used by existing tools, namely memory-efficient (out-of-core) graph heuristics, which he implemented in the netclust open-source software. 

In 2009 Arnold obtained his doctorate degree in Bioinformatics and continued, as post-doc in the same lab, in the development of the multi-parametric version of this tool called multi-netclust, which enabled analysis of combined data networks (e.g., protein similarity networks) from different sources. In 2011 Arnold moved to Switzerland where he joined the Department of Ecology and Evolution, University of Lausanne and Swiss Institute of Bioinformatics, to work on an e-Science project (Grid-enabled Selectome) aimed at speeding up the detection of positive selection in animal genomes using the Swiss Multi Science Grid (SMSCG).

In 2012 Arnold joined the Department of Genetics at the Erasmus Medical Center in Rotterdam, where he focused on computational aspects of the semi-quantitative mass-spectrometry(MS)-based proteomics, in particular on reliable detection of cellular responses (pathways) upon exposure of mammalian cells to different non-ionizing electromagnetic fields. To facilitate the MS data handling and statistical analyses, Arnold developed PIQMIe, a freely available proteomics web server.

Arnoldâ€™s work has been centered around the development of efficient algorithms and user-friendly (web-based) tools for scalable molecular data mining using distributed/parallel computing infrastructures. In 2015 Arnold joined the Netherlands eScience Center where he works on the application of semantic web technologies on data integration and knowledge discovery in life sciences.
